server/chat-plugins/othermetas.js

```diff
@@ -339,6 +339,28 @@ const commands = {
 		this.sendReply(`|raw|${Chat.getDataPokemonHTML(template)}`);
 	},
 	scalemonshelp: [`/scale OR /scalemons <pokemon> - Shows the base stats that a Pokemon would have in Scalemons.`],
+
+	'!natureswap': true,
+	ns: 'natureswap',
+	natureswap(target, room, user) {
+		if (!this.runBroadcast()) return;
+		let nature = target.trim().split(' ')[0];
+		let pokemon = target.trim().split(' ')[1];
+		if (!toID(nature) || !toID(pokemon)) return this.parse(`/help natureswap`);
+		let natureObj = Dex.getNature(nature);
+		if (!natureObj.exists) return this.errorReply(`Error: Pokemon ${natureObj} not found.`);
+		let template = Dex.deepClone(Dex.getTemplate(pokemon));
+		if (!template.exists) return this.errorReply(`Error: Pokemon ${pokemon} not found.`);
+		// @ts-ignore
```

Comment:
You should not require `@ts-ignore` at all here, please fix the type errors instead.